demos-europe/demosplan-core
# feat(refs DPLAN-3796): Add input for file name in exports

### Ticket
DPLAN-3796

Add input for file name for docx exports. You can use {ID}, {NAME} and {EINGANGSNR} as placeholders. 



### How to review/test
Verfahren -> Stellungnahmen -> Exportieren -> einzelne DOCX-Dateien
Verfahren -> Stellungnahmen -> Details und Erwiderungen -> Exportieren
Try to use the placeholders and export. Also see if example file name beneath the input shows the right file name


### Linked PRs (optional)
https://github.com/demos-europe/demosplan-core/pull/3395
